lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -253,7 +253,7 @@ def define
 
       outcome :outcome_opposite_sex_marriage_consular_cni_residing_in_third_country
 
-      outcome :outcome_os_consular_cni
+      outcome :outcome_opposite_sex_marriage_consular_cni_residing_in_uk_or_ceremony_country
```

Comment:
The `_ceremony_country` suffix makes more sense here, because the outcome is not specific to an individual ceremony country.